idowuadeleke/BANKA
# #165665489 fetch backend api

#### What does this PR do?
use vanilla javascript to fetch backend API

### Description of Task to be completed?
implement fetch for sign up and sign in

### How should this be manually tested?
- clone the repo `git clone https://github.com/idowuadeleke/BANKA.git`

- on your terminal run `cd BANKA`

- run `git checkout ch-165665489-build-out-frontend`

- run `npm install`

- run `npm run start`

- open index.html and confirm backend to frontend integration
  NB see screenshot below on the request data to pass

### Any background context you want to provide?
NA
### What are the relevant pivotal tracker stories?
[#165665489](https://www.pivotaltracker.com/story/show/165665489)

### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/40268693/56871072-8f6fca80-6a10-11e9-8802-a5293a38c870.png)
![image](https://user-images.githubusercontent.com/40268693/56871083-aa423f00-6a10-11e9-8d0f-ee4b07b71b59.png)
